src/manifest/dbtProject.ts

```diff
@@ -516,6 +517,96 @@ export class DBTProject implements Disposable {
     return output.compiled_sql;
   }
 
+  public async validateSql(request: {
```

Comment:
```suggestion
 async validateSql(request: {
```

Please remove public modifiers.